### DISCUSSION TITLE
VTX&camera doesn't work with crazyflie electric power

### DISCUSSION THREAD

**Discussion by allegorywrite** [bitcraze/discussions#1153] [Q&A] [answered]
I'm trying to create an FPV expansion deck using vtx.
I found [this article](https://www.bitcraze.io/2022/11/fpv-deck-hack/?unapproved=38619&moderation-hash=8093f31add4a144e3c8b1e559a687f8a#comment-38619), and made a drone expansion deck with
1. Runcam Atom-W
2. vtx(tbs unify pro32 nano 5g8)
3.3vDC-5vDC converter
4. 250mAh LiPo battery(default)
5. thrust upgrade propeller and motor

However, when the motor rotates, the VTX shuts down due to insufficient current or voltage, making the camera and drone incompatible. In particular, when I tried to supply power from VCOM, the camera did not start even when stopped. About crazyflie or lipo battery, is there anything I can do to resolve this issue? Any tips are welcome.

**Comment by knmcguire** [comment, marked as answer]
hi! seems that your question was answered as part of the blog comments: https://www.bitcraze.io/2022/11/fpv-deck-hack/?unapproved=38619&moderation-hash=8093f31add4a144e3c8b1e559a687f8a#comment-38619

**Comment by allegorywrite** [comment]
Yes! The question in that article solved my problem. Sorry for the duplicate question.
